mindmaps-graph/src/main/java/io/mindmaps/graph/internal/AbstractMindmapsGraph.java

```diff
@@ -364,6 +364,11 @@ public Instance getInstance(String id) {
     }
 
     @Override
+    public Instance getInstance(Resource uniqueResource){
+        return uniqueResource.owner();
+    }
```

Comment:
Do we really need this if it only calls a method on the resource passed in?
We should probably be careful when adding more stuff to `MindmapsGraph`.